python/rules_engine.py

```diff
@@ -2644,6 +2643,136 @@ def v708(self):
+		edit_name = "v712"
+		fail_df = self.lar_df[((self.lar_df.loan_costs=="Exempt") | (self.lar_df.points_fees=="Exempt")) & ((self.lar_df.loan_costs!="Exempt")
+					| (self.lar_df.points_fees!="Exempt"))]
+		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
+
+	def v713_1(self):
+		"""If the Automated Underwriting System exemption
+			election is taken,
+			1) Automated Underwriting System: 1 and
+			Automated Underwriting System Result: 1 must be
+			reported 1111; and"""
+		field = "Automated Underwriting System"
+		edit_name = "v713_1"
+		fail_df = self.lar_df[(self.lar_df.aus_1=="1111") & ((self.lar_df.aus_1!="1111") & (self.lar_df.aus_result_1!="1111"))]
+		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
+	
+	def v713_2(self):
+		"""If the Automated Underwriting System exemption
+			election is taken,
+			2) Automated Underwriting System: 2, Automated
+			Underwriting System: 3, Automated Underwriting
+			System: 4, Automated Underwriting System: 5,
+			Automated Underwriting System: Conditional Free
+			Form Text Field for Code 5, Automated Underwriting
+			System Result: 2, Automated Underwriting System
+			Result: 3, Automated Underwriting System Result: 4,
+			Automated Underwriting System Result: 5, and
+			Automated Underwriting System Result: Conditional
+			Free Form Text Field for Code 16 must be left blank."""
+		field = "Automated Underwriting System"
+		edit_name = "v713_2"
+		fail_df = self.lar_df[(self.lar_df.aus_1=="1111") & ((self.lar_df.aus_2!="") & (self.lar_df.aus_3!="") & (self.lar_df.aus_4!="") & 
+			(self.lar_df.aus_5!="") & (self.lar_df.aus_code_5!="") & (self.lar_df.aus_result_2!="") & (self.lar_df.aus_result_3!="") 
+			& (self.lar_df.aus_result_4!="") & (self.lar_df.aus_result_5!="") & (self.lar_df.aus_code_16!=""))]
+		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
+
+	def v714(self):
+		"""1) If the Application Channel exemption election is
+			taken, Submission of Application and Initially Payable
+			to Your Institution must be reported 1111."""
+		field = "Application Channel"
+		edit_name = "v714"
+		fail_df = self.lar_df[((self.lar_df.app_submission == "1111") & (self.lar_df.initially_payable != "1111"))|
+		((self.lar_df.app_submission != "1111") & (self.lar_df.initially_payable == "1111"))]  
+		#((self.lar_df.app_submission != "1111") & (self.lar_df.initially_payable != "1111"))]
+		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
+	
+	def v715(self):
+		"""1) If the Non-Amortizing Features exemption election
+			is taken, Balloon Payment, Interest-Only Payments,
+			Negative Amortization and Other Non-amortizing
+			Features must be reported 1111."""
+		field = "Non-Amortizing Features"
+		edit_name = "v715"
+		fail_df = self.lar_df[(
+			(self.lar_df.non_amort_features=="1111") | (self.lar_df.balloon=="1111") | (self.lar_df.int_only_pmts=="1111")|
```

Comment:
I think this logic might need a recheck. I'm seeing if any are = 1111 and all are = 1111 then fail.